src/ResourceManager/PolicyInsights/ChangeLog.md

```diff
@@ -19,6 +19,11 @@
 -->
 ## Current Release
 
+## Version 1.0.0
```

Comment:
remove this... the version number will be added by the versioning tool, just leave the latest changes under Current Release